### PR TITLE
fixed documentation for docs-agent demo

### PR DESCRIPTION
**Pull request by thesanju**
## Description of the change
fixed bug in documentation which was causing a issue while running the Docs Agent chat app

## Motivation
this bug was causing me issue https://github.com/google/generative-ai-docs/issues/176 and I added a extra line in documentation which solves this issue and @nickvander suggested me to fix this in documentation

## Type of change
Documentation

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have added detailed comments to my code where applicable.
- [x] I have verified that my change does not break existing code.
- [x] My PR is based on the latest changes of the main branch (if unsure, please run `git pull --rebase upstream main`).
- [x] I am familiar with the [Google Style Guide](https://google.github.io/styleguide/) for the language I have coded in.
- [x] I have read through the [Contributing Guide](https://github.com/google/generative-ai-docs/blob/main/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://cla.developers.google.com/about).

### CODE REVIEW

File: demos/palm/python/docs-agent/README.md [hunk 1]
```diff
@@ -601,6 +601,10 @@ To launch the Docs Agent chat app, do the following:
    **Note**: The Docs Agent chat app runs on port 5000 by default. If you have an application
    already running on port 5000 on your host machine, you can use the `-p` flag to specify
    a different port (for example, `poetry run ./chatbot/launch.sh -p 5050`).
+   
+   **Note**: If this ```poetry run ./chatbot/launch.sh``` command fails to run, check the ```HOSTNAME```
```
Comment (kyolee415) [inline demos/palm/python/docs-agent/README.md:605]: ``` needs to be `

For instance, `poetry run ./chatbot/launch.sh` and `HOSTNAME` 

Very close! Thanks!

Comment (kyolee415) [reply demos/palm/python/docs-agent/README.md:605]: Have you added this change? I don't currently see it on this pull request.

Comment (thesanju) [reply demos/palm/python/docs-agent/README.md:605]: am sorry I did't get what are you saying can you be more specific ?

Comment (thesanju) [reply demos/palm/python/docs-agent/README.md:605]: sorry my bad, I created a new PR please check it, thank you

**Pull request closed without merging**